assets/js/components/Dashboard/Dashboard.tsx

```diff
@@ -17,11 +17,12 @@ const Dashboard: ComponentType = () => {
   const [bannerDone, setBannerDone] = useState(false);
 
   useEffect(() => {
-    fetchAlerts((newAlerts, screens_by_alert) => {
+    fetchAlerts((all_alert_ids, newAlerts, screens_by_alert) => {
```

Comment:
Can you change `newAlerts` to snake case to match the other two parameters? I think I'm the one to introduce the inconsistency, so sorry about that 😬

Reply:
Actually going to use camelCase because it's a tsx file. Thanks for the catch!